Octane at EmberConf (#479)
ember-learn/ember-blog: source/2020-01-10-the-ember-times-issue-130.md

---

> ## [Octane at EmberConf 2020 📢](https://emberconf.com)

Reviewer: I added a new line after the header. I think this will help pass Travis CI.
```suggestion
## [Octane at EmberConf 2020 📢](https://emberconf.com)

```

With [Ember Octane](https://blog.emberjs.com/2019/12/20/octane-is-here.html) released it is not a surprise that it’s also a hot topic for [EmberConf 2020](https://emberconf.com/#/speakers). 

> If you want to hear [people](https://emberconf.com/#/speakers) talk about Octane or talk with people about Octane, EmberConf in Portland, OR is the place to be. The 16th till 18th of March will be filled with awesome talks about Octane by [Godfrey Chan](https://github.com/chancancode), [Suchita Doshi](https://github.com/suchitadoshi1987), [DanMonroe](https://github.com/DanMonroe) and probably some others. 

Reviewer: To shorten this paragraph a bit, what do you think about:
```suggestion
If you want to [listen and talk to people](https://emberconf.com/#/speakers) about Octane, EmberConf in Portland, OR is the place to be. The 16th till 18th of March will be filled with awesome talks by [Godfrey Chan](https://github.com/chancancode), [Suchita Doshi](https://github.com/suchitadoshi1987), [DanMonroe](https://github.com/DanMonroe) and many others. 
```
?

In addition, can we add an extra line between lines 89 and 90, and lines 90 and 91? I think, currently, when the Markdown file gets converted to the blog post, we will see one big paragraph instead of 3 paragraphs (I think the latter is what you wanted?).

> It’s of course also a nice place to meet up with your Ember-friends and learn about many more topics. For more information check the [website](https://emberconf.com) or join the #ember-conf channel on Discord.

Reviewer: I added EmberConf to give context (to be more accessible, possibly).
```suggestion
It’s of course also a nice place to meet up with your Ember-friends and learn about many more topics. For more information, check the [EmberConf website](https://emberconf.com) and join the #ember-conf channel on Discord.
```

> <change section title emoji>

Reviewer: Can you remove this placeholder?

---

## [Octane at EmberConf 2020 📢](https://emberconf.com)

With [Ember Octane](https://blog.emberjs.com/2019/12/20/octane-is-here.html) released it is not a surprise that it’s also a hot topic for [EmberConf 2020](https://emberconf.com/#/speakers). 

If you want to [listen and talk to people](https://emberconf.com/#/speakers) about Octane, EmberConf in Portland, OR is the place to be. The 16th till 18th of March will be filled with awesome talks by [Godfrey Chan](https://github.com/chancancode), [Suchita Doshi](https://github.com/suchitadoshi1987), [DanMonroe](https://github.com/DanMonroe) and many others. 

> It’s of course also a nice place to meet up with your Ember-friends and learn about many more topics. For more information, check the [EmberConf website](https://emberconf.com) and join the #ember-conf channel on Discord.

Reviewer: Regarding Alex linting, I think we can omit the phrase "of course" here, and the sentence will still flow nicely and sounds friendly. Would that be okay with you?
```suggestion
It’s also a nice place to meet up with your Ember-friends and learn about many more topics. For more information, check the [EmberConf website](https://emberconf.com) and join the #ember-conf channel on Discord.
```
